decoder/pom.xml

```diff
@@ -57,7 +57,8 @@
 
 			<plugin>
 				<groupId>com.mycila</groupId>
-				<artifactId>license-maven-plugin</artifactId>
+				<artifactId>maven-license-plugin</artifactId>
+                <version>${maven.license.plugin.version}</version>
```

Comment:
The indentation looks wrong here.

Reply:
addressed in 1fe5940d667da0881ad8595be524a396dc69a0e0